x/deployment/ante/ante.go

```diff
@@ -0,0 +1,47 @@
+package ante
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/ovrclk/akash/x/deployment/keeper"
+	"github.com/ovrclk/akash/x/deployment/types"
+)
+
+type DeploymentDecorator struct {
```

Comment:
this can be private.  also, it probably doesn't need to be specific to deployment-qualified:

```go
type anteDecorator struct {
  keeper keeper.Keeper
}
```